assets/js/scroll-to-top-button.js

```diff
@@ -0,0 +1,21 @@
+// Get the button:
```

Comment:
```suggestion
/* eslint-env browser */

// Get the button:
```

Reply:
What does that do? :stuck_out_tongue:

Reply:
eslint is the linter.

i think this comment will tell eslint that this javascript runs in a browser, so it is okay to expect `window` and `document` to exist without defining them.

Reply:
:smile: